rs/registry/canister/src/registry_lifecycle.rs

```diff
@@ -58,6 +66,88 @@ pub fn canister_post_upgrade(
     }
 }
 
+/*
+Three node operators had corrupted NodeOperatorRecords which caused issues during migration (https://dashboard.internetcomputer.org/proposal/139210).
```

Comment:
Please, explain (in comments) not just what the bad result was. Explain how did these bad results occur so that people can understand why this fix is correct.